2-mandatory/4-restaurant.js

```diff
@@ -21,6 +21,15 @@ const MENU = {
 
 let cashRegister = {
```

Comment:
Great solution! Good job using a reference to the prices stored in the MENU object